src/url-mapper.js

```diff
@@ -140,6 +140,11 @@ export default class UrlMapper {
   _addMemoryCopy(mappedUrl) {
     this._removeMemoryCopy(mappedUrl.url);
 
+    // ignore broken url mappings
+    if(!mappedUrl.url) {
```

Comment:
This solves app-startup, when the mappings are read from the db and put into memory.
However, this won't stop new mappings from being added to the db. I mean, in memory, we'll _never_ have a broken mapping, but the db will fill up with broken mappings that are never used